detection-rules/attachment_callback_phish_with_img.yml

```diff
@@ -10,48 +10,47 @@ severity: "high"
 source: |
   type.inbound
   and (
-        (
-            sender.email.domain.root_domain in $free_email_providers
-            and sender.email.email not in $recipient_emails
-        )
-        or (
-            sender.email.domain.root_domain not in $free_email_providers
-            and sender.email.domain.domain not in $recipient_domains
-        )
+    (
+      sender.email.domain.root_domain in $free_email_providers
+      and sender.email.email not in $recipient_emails
+    )
+    or (
+      sender.email.domain.root_domain not in $free_email_providers
+      and sender.email.domain.domain not in $recipient_domains
+    )
   )
-
   and sender.email.domain.root_domain in $free_email_providers
-  and any(attachments, .file_extension in~ ('png', 'jpg', 'jpeg')
-      and any(file.explode(.),
-          length(filter(.scan.strings.strings, strings.ilike(.,
-                "*purchase*",
-                "*subscription*",
-                "*antivirus*",
-                "*order*",
-                "*support*",
-                "*receipt*",
-                "*amount*",
-                "*charged*",
-                "*invoice*",
-                "*call*",
-                "*cancel*",
-                "*renew*",
-                "*refund*",
-                "*+1*"
-            ))) >= 4)
-
-      and any(file.explode(.),
-          length(filter(.scan.strings.strings, strings.ilike(.,
-                "*geek squad*",
-                "*lifelock*",
-                "*best buy*",
-                "*mcafee*",
-                "*norton*",
-                "*ebay*",
-                "*paypal*",
-                "*secure anywhere*"
-            ))) >=1)
-            )
+  and any(attachments,
+          .file_type in~ ('png', 'jpg')
+          and any(file.explode(.),
+                  4 of (
```

Comment:
We chatted about this one in Slack. The main distinction is that we want to know that at least 4 of our substrings were found. The previous approach checks for how many strings return from strelka match at least one.

If `.scan.strings.strings` was `["to purchase an antivirus subscription now, call or cancel now"]`, then we wouldn't match before because everything was in the same string. Alternatively, if we you had `["+1", "+1", "+1", "+1"]`, that used to match before because it didn't check if the same thing was matched.

Ultimately it just means we were counting the wrong thing and it was inverted, so this is one way to flip it.